<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meGlass</class>
 <widget class="QWidget" name="meGlass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Glass type</string>
     </property>
     <property name="class" stdset="0">
      <string>MatProperty</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QRadioButton" name="architecturalGlass">
        <property name="text">
         <string>Architectural</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="standardGlass">
        <property name="text">
         <string>Standard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="volumetricGlass">
        <property name="text">
         <string>Volumetric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="frostedGlass">
        <property name="text">
         <string>Frosted</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ReTextureAvatar" name="transmission">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="caption">
      <string>Tint</string>
     </property>
     <property name="ID">
      <string>Kt</string>
     </property>
     <property name="breadcrumb">
      <string>Transmission</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ReTextureAvatar" name="reflection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="caption">
      <string>Reflection</string>
     </property>
     <property name="ID">
      <string>Kr</string>
     </property>
     <property name="breadcrumb">
      <string>Reflection</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="iorSettings">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frIORSettings">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>IOR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="ior">
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="presetName">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="iorPresets">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="animated">
         <bool>false</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Preset</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IOR</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="gbThinFilm">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Thin film</string>
       </property>
       <property name="class" stdset="0">
        <string>MatProperty</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frIOR">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>IOR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="thinFilmIOR"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frThickness">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Thickness</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="thinFilmThickness">
             <property name="maximum">
              <double>2500.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="gbRoughness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Roughness</string>
       </property>
       <property name="class" stdset="0">
        <string>MatProperty</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frhRoughness">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Horizontal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="uRoughness">
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnAnisotropic">
          <property name="text">
           <string>L</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="vRoughness_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Vertical</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="vRoughness">
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="dispersion">
       <property name="text">
        <string>Dispersion</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QFrame" name="frCauchy">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Cauchy B-coefficient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cauchyB">
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QFrame" name="frCauchy_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Clarity at depth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="clarity">
          <property name="suffix">
           <string>m</string>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReTextureAvatar</class>
   <extends>QWidget</extends>
   <header>RealityUI/ReTextureAvatar.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ior</tabstop>
  <tabstop>thinFilmIOR</tabstop>
  <tabstop>thinFilmThickness</tabstop>
  <tabstop>uRoughness</tabstop>
  <tabstop>vRoughness</tabstop>
  <tabstop>dispersion</tabstop>
  <tabstop>cauchyB</tabstop>
  <tabstop>iorPresets</tabstop>
  <tabstop>frostedGlass</tabstop>
  <tabstop>architecturalGlass</tabstop>
  <tabstop>standardGlass</tabstop>
  <tabstop>btnAnisotropic</tabstop>
  <tabstop>volumetricGlass</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
